cmd/support-cron-scheduler/res/configuration.yaml

```
@@ -0,0 +1,29 @@
+Writable:
+  LogLevel: INFO
+  InsecureSecrets:
+    DB:
+      SecretName: "redisdb"
+      SecretData:
+        username: ""
+        password: ""
+    POSTGRES:
+      SecretName: "postgres"
+      SecretData:
+        username: "postgres"
+        password: "postgres"
+Service:
+  Host: localhost
+  Port: 59863
+  StartupMsg: "This is the Support Cron Scheduler Microservice"
+
+MessageBus:
+  Optional:
+    ClientId: support-cron-scheduler
+
+Database:
+  Host: "localhost"
+  Port: 5432
+  Timeout: "5s"
+  Type: "postgres"
```

```suggestion
  Type: postgres
```

for configuration.yaml, don't quote for string config
it can be eliminated